js/routecontroller.js

```diff
@@ -108,9 +108,8 @@ define(function(require) {
 
 			var folder = account.get('folders').get(folderId);
 			if (_.isUndefined(folder)) {
-				// TODO: show first folder of this account
-				_this.default();
-				return;
+				folder = account.get('folders').at(0);
```

Comment:
Note: you could also use [``.first()``](http://backbonejs.org/#Collection-Underscore-Methods), but ``.at(0)`` is also fine as it does the same thing ;-)